### PR TITLE
Added pushover notifier

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added pushover notifier
- fixed default path

### CODE REVIEW

File: core/http.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package core
+
+// IsStatusCodeIs2xx returns true if the provided status code is not 2xx
+func IsStatusCodeIs2xx(statusCode int) bool {
```
Comment (andreibancioiu) [inline core/http.go:4]: Name does not match description. Also, perhaps there's a typo in the name?

File: notifiers/pushoverNotifier.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+// NewPushoverNotifier will create a new Pushover notifier
+func NewPushoverNotifier(url string, token string, userKey string) *pushoverNotifier {
+	return &pushoverNotifier{
+		httpClientWrapper: httpSDK.NewHttpClientWrapper(nil, url),
+		token:             token,
+		userKey:           userKey,
+	}
+}
+
+// OutputMessages will push the provided messages as error
```
Comment (andreibancioiu) [inline notifiers/pushoverNotifier.go:47]: Not necessarily `as error`, right?

File: notifiers/pushoverNotifier.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+func (notifier *pushoverNotifier) OutputMessages(messages ...core.OutputMessage) {
+	log.Debug("notifier.OutputMessage pushing error notification", "num messages", len(messages))
+	if len(messages) == 0 {
+		return
+	}
+
+	msgString := ""
+	maxMessageOutputType := core.MessageOutputType(0)
```
Comment (andreibancioiu) [inline notifiers/pushoverNotifier.go:55]: Maybe `max` can also be `highest` (just an opinion, maybe `max` is already descriptive enough).

File: notifiers/pushoverNotifier.go [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+		}
+
+		msgString += createMessageString(msg)
+	}
+
+	title := createTitle(maxMessageOutputType, messages[0].ExecutorName)
+
+	err := notifier.pushNotification(msgString, title)
+	if err != nil {
+		log.Error("notifier.OutputMessage pushing notification", "error", err)
```
Comment (andreibancioiu) [inline notifiers/pushoverNotifier.go:68]: Typo / name of function is `OutputMessages`.

File: notifiers/pushoverNotifier_test.go [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+
+func TestPushoverNotifier_FunctionalTest(t *testing.T) {
+	t.Skip("this is a functional test, will need real credentials")
+
+	_ = logger.SetLogLevel("*:DEBUG")
+
+	notifier := NewPushoverNotifier(
+		"https://api.pushover.net/1/messages.json",
+		"", // TODO: replace here your token value
+		"", // TODO: replace here your userKey value
```
Comment (andreibancioiu) [inline notifiers/pushoverNotifier_test.go:306]: Maybe get them using `os.Getenv`?

Comment (iulianpascalau) [reply notifiers/pushoverNotifier_test.go:306]: very very interesting approach. :+1:

File: core/http.go (hunk 1 above)
Comment (iulianpascalau) [reply core/http.go:4]: refactored

File: notifiers/pushoverNotifier.go (hunk 2 above)
Comment (iulianpascalau) [reply notifiers/pushoverNotifier.go:47]: right, refactored. Remained from previous tests/tries

File: notifiers/pushoverNotifier.go (hunk 3 above)
Comment (iulianpascalau) [reply notifiers/pushoverNotifier.go:55]: right, refactored

File: notifiers/pushoverNotifier.go (hunk 4 above)
Comment (iulianpascalau) [reply notifiers/pushoverNotifier.go:68]: fixed

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

File: notifiers/logNotifier.go [hunk 6]
```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	"multiversx/mx-chain-keys-monitor-go/core"
+	"github.com/multiversx/mx-chain-keys-monitor-go/core"
 
 	"github.com/multiversx/mx-chain-core-go/core/check"
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply notifiers/logNotifier.go:9]: will do in the next PR

**Pull request merged**